kong/api/routes/certificates.lua

```diff
@@ -161,14 +279,12 @@ return {
       return helpers.responses.send_HTTP_OK(row)
     end,
 
-
     PATCH = function(self, dao_factory, helpers)
       return crud.patch(self.params, dao_factory.ssl_certificates, {
         id = self.ssl_certificate_id
       })
     end,
 
-
```

Comment:
I don't think those changes are desired - this is intended style (2 lines jump between semantically similar code blocks) that we are enforcing in newer parts of the codebase.

Reply:
There are ~5/6 similar other changes in this patchset - will you revert them? Thanks!